index.html

```diff
@@ -1,26 +1,36 @@
-<DOCTYPE html>
+<!DOCTYPE html>
 <html>
     <head>
-        <meta charset="utf-8">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Calculator</title>
         <link rel="stylesheet" href="css/style.css" type="text/css">
-        <script src="./js/script.js"></script>
-    </head>
+    </head> 
     <body>
         <div class="calculator">
-            <form class="calculator-form">
-                <div class="calculator-form__input">
-                    <input type="text" id="a" value="1">
-                    <input type="text" id="b" value="1"> 
-                </div>
-                <div class="calculator-form__action-bar action-bar"> 
-                   <input type="button" class="action-bar__button action-bar__button-sum" id="calc-sam" onclick="calcSum();" value="Sum">
-                   <input type="button" class="action-bar__button action-bar__button-sub" id="calc-sub" onclick="calcSub();" value="Sub">
-                   <input type="button" class="action-bar__button action-bar__button-mul" id="calc-mul" onclick="calcMul();" value="Mul">
-                   <input type="button" class="action-bar__button action-bar__button-div" id="calc-div" onclick="calcDiv();" value="Div">
-                </div>
-            </form>
-            <div class="calculator-result" id="result" ></div>
+            <div class="calculator-form action-bar">
+                <input type="text" class="calculator-form__item action-bar__output" id="calc-output"  disabled>
+                <input type="button" class="calculator-form__item action-bar__button-ce" id="CE" value="CE">
+                <input type="button" class="calculator-form__item number action-bar__button-seven" id="seven" value="7">
+                <input type="button" class="calculator-form__item number action-bar__button-eight" id="eight" value="8">
+                <input type="button" class="calculator-form__item number action-bar__button-nine" id="nine" value="9">
+                <input type="button" class="calculator-form__item operators action-bar__button-div" id="div" value="&divide">
+                <input type="button" class="calculator-form__item number action-bar__button-four" id="four" value="4">
+                <input type="button" class="calculator-form__item number action-bar__button-five" id="five" value="5">
+                <input type="button" class="calculator-form__item number action-bar__button-six" id="six" value="6">
+                <input type="button" class="calculator-form__item operators action-bar__button-mul" id="mul" value="&times">
```

Comment:
As I can see, `operators` class is not used in `styles.scss` file so you can safely remove it.